### PR TITLE
Add remaining Otel (0.7.0) semantic attributes

### PR DESCRIPTION
**Pull request by slashmo**
This PR is part of the effort to be on par with Otel 0.7.0 (#32). It adds the remaining semantic span attributes and updates existing ones. All documentation comments pointing towards Otel have been updated to point to the specific 0.7.0 version on GitHub.

### CODE REVIEW

**Comment by slashmo** [comment]
@ktoso / @tomerd: I'm not sure why the CI is failing here, seems to be something wrong with Jenkins:
> `java.io.FileNotFoundException: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.275.b01-0.el7_9.x86_64/jre/lib/currency.data (No such file or directory)`

**Comment by ktoso** [comment]
@swift-server-bot test this please

File: Sources/TracingOpenTelemetrySupport/SpanAttribute+DatabaseSemantics.swift [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+extension SpanAttributeName {
+    /// - See: DatabaseAttributes
+    public enum Database {
+        /// - See: DatabaseAttributes
+        public static let system = "db.system"
+        /// - See: DatabaseAttributes
+        public static let connectionString = "db.connection_string"
+        /// - See: DatabaseAttributes
+        public static let user = "db.user"
+        /// - See: DatabaseAttributes
```
Comment (ktoso) [inline Sources/TracingOpenTelemetrySupport/SpanAttribute+DatabaseSemantics.swift:25]: the keyword is `- SeeAlso: ` AFAIK, do we have it wrong everywhere else too?

File: Sources/TracingOpenTelemetrySupport/SpanAttribute+EndUserSemantics.swift [hunk 2]
```diff
@@ -40,7 +40,7 @@ extension SpanAttributes {
 
 /// End-user-related semantic conventions as defined in the OpenTelemetry spec.
 ///
-/// - SeeAlso: [OpenTelemetry: General identity attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/b70565d5a8a13d26c91fb692879dc874d22c3ac8/specification/trace/semantic_conventions/span-general.md#general-identity-attributes) (as of August 2020)
+/// - SeeAlso: [OpenTelemetry: General identity attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/v0.7.0/specification/trace/semantic_conventions/span-general.md#general-identity-attributes)
```
Comment (ktoso) [inline Sources/TracingOpenTelemetrySupport/SpanAttribute+EndUserSemantics.swift:43]: 🎉

**Review by ktoso — APPROVED** [review]
LGTM, thanks! 

Can you mass change the `- See` to `- SeeAlso` and merge? 👍 

CI just had some hiccups, builds ok now :-)

**Comment by ktoso** [comment]
Does this completely resolve https://github.com/apple/swift-distributed-tracing/issues/32 or still some things to check?

**Comment by slashmo** [comment]
> Does this completely resolve #32 or still some things to check?
> @ktoso

I think it depends on whether we want to add some kind of `Resource` (#39). If so, I'd include it in #32 as well. Otherwise, I'd have to look through the spec once more but I think we're already on par with Otel 0.7.0.

**Pull request merged**